File: applications/grided/server/src/org/monet/grided/core/services/space/data/Service.java
package org.monet.grided.core.services.space.data;

import java.io.Serializable;

public class Service implements Serializable {
    private static final long serialVersionUID = 7717877466166391128L;
    private String name;
    private ServiceType type;

    public Service(String name, ServiceType type) {
        this.name = name;
        this.type = type;
    }
    
    public String getName() {
        return this.name;
    }
    
    public ServiceType getType() {
        return this.type;
    }
}

File: applications/space/client/setup/src/core/model/service.js
function Service(data) {   
  this.id = "";  
  this.code = "";
  this.name = "";
  this.type = "";
  this.enabled = false; 

  if (data != null) { 
	this.copyFrom(data); 
  }
}

//---------------------------------------------------------------------
Service.prototype.copyFrom = function(data) {    
  this.id = data.id;
  this.code = data.code;
  this.name = data.name;
  this.type = data.type;
  this.enabled = data.enabled;
};

